draft/index.html

```diff
@@ -255,21 +255,71 @@ <h2>Core Concepts</h2>
+          <dfn data-lt="text">String</dfn>
+          - This is a sequence of zero or more Unicode characters. Strings are written within double quotes and can contain various escape characters.
+        </li>
+        <li>
+          <dfn data-lt="number">Integer</dfn>
+          - This is a signed decimal number (ex. 20) that may contain a fractional part and may use exponential E notation. JSON does not support octal or hexadecimal numbers. 
+        </li>
+        <li>
+          <dfn data-lt="list">Array</dfn>
+          - This is an ordered collection of values. An array begins with <code>[</code> (left bracket) and ends with <code>]</code> (right bracket). Values are separated by <code>,</code> (comma).
+        </li>
+        <li>
+          <dfn>Boolean</dfn>
+          - This is either <code>true</code> or <code>false</code>.
+        </li>
+
+        <li>
+          <!-- "required" is not a defined term since it is a default and unused currently.
+               So, we just make it bold italics to provide a similar style as defined terms.
+          -->
+        ( <b><i>required</i></b> ) - is the default and might be shown at times to signify those fields which are mandatory and MUST be used in all reconciliation services.
+        If the field is not specified as either ( <b>required</b> ) or ( <b>optional</b> ), then it is REQUIRED.
+        </li>
+        <li>
+        (<dfn
+          data-lt="NOT MANDATORY|NON-MANDATORY">
+          optional
+        </dfn>) - is used to signify those fields which are not mandatory and MAY be used.
+        </li>
+      </ul>
+      </p>
+      </p>
       <section>
         <h3 data-link-for="entities">Entities</h3>
         <p>
            An <dfn>entity</dfn> is a record in the data source exposed by the service.
            It comprises the following fields:
-           <dl>
-	     <dt><code>id</code></dt>
-             <dd>an identifier, which is a non-empty string. This identifier must be unique among all entities;</dd>
-             <dt><code>name</code></dt>
-             <dd>a <emph>name</emph>, which is also a non-empty string;</dd>
-             <dt><code>description</code></dt>
-             <dd>an optional <emph>description</emph> as a human-readable string;</dd>
-             <dt><code>type</code></dt>
-             <dd>an array of <a>types</a>, possibly empty;</dd>
-           </dl>
+        <dl>
+        <dt><code>id</code></dt>
+          <dd>{{String}} a non-empty identifier. This identifier must be unique among all entities;</dd>
+        <dt><code>name</code></dt>
+          <dd>{{String}} ({{optional}}) a non-empty <emph>name</emph> or <emph></emph>label</emph> for the entity;</dd>
+        <dt><code>description</code></dt>
+          <dd>{{String}} ({{optional}}) a <emph>description</emph> as a human-readable string;</dd>
+        <dt><code>type</code></dt>
+          <dd>{{Array}} ({{optional}}) list of <a>types</a>, possibly empty, the entity is classified by;</dd>
```

Comment:
@fsteeg Does this appended phrase make things more clear, or does it assume too much?  I felt like it was missing this context, which might not be apparent to developers outside of Semantic Web or Knowledge Graph circles (or just getting introduced to them) when we're talking about Entities and their Types.